docs/topics/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -0,0 +1,78 @@
+[#proc_deploying-an-application-to-openshift]
+= Deploying an Application to OpenShift
+
+This procedure shows you how to:
+
+* Build your {SpringBoot} application from source and create an application image using the S2I build process.
```

Comment:
It technically uses S2I (FMP uses S2I binary source builds), but from user's point of view, S2I is something completely different. I wouldn't mention S2I here.